lib/puppet/module_tool/applications/builder.rb

```diff
@@ -2,6 +2,7 @@
 require 'json'
 require 'puppet/file_system'
 require 'pathspec'
+require 'facter'
```

Comment:
I'm not sure if we should be explicitly requiring facter... @branan @MikaelSmith do you have concerns?

Reply:
If anyone uses module tool as part of their tests, this might not work as expected with Facter 2. In general it doesn't seem problematic though, we `require 'facter'` quite a few places.

Reply:
It's also probably not necessary to do it explicitly, as `puppet.rb` does so and is usually the entry point to everything.

Reply:
@MikaelSmith brings up a good point, if we rely on facter, then we'll abort when using facter 2 as is common in CI pipelines. Thoughts @ahpook?

Reply:
Well, I think `Facter.value(:fips_enabled` will return false, meaning we'll never raise. That seems like ok behavior in this case.